models/issue_watch.go

```diff
@@ -15,14 +15,12 @@ import (
 type IssueWatchMode int8
 
 const (
-	// IssueWatchModeNone don't watch
-	IssueWatchModeNone IssueWatchMode = iota // 0
-	// IssueWatchModeNormal watch issue (from other sources)
-	IssueWatchModeNormal // 1
-	// IssueWatchModeDont explicit don't auto-watch
-	IssueWatchModeDont // 2
-	// IssueWatchModeAuto watch issue (from AutoWatchOnChanges)
-	IssueWatchModeAuto // 3
+	// IssueWatchModeNormal watch issue
+	IssueWatchModeNormal IssueWatchMode = iota // 0
```

Comment:
Even if we don't have `IssueWatchModeNone` in the database, I think It's still useful to match the values from `repo_watch`:
```
    // RepoWatchModeNone not watching
    RepoWatchModeNone RepoWatchMode = iota // 0
    // RepoWatchModeNormal watch repository (from other sources)
    RepoWatchModeNormal // 1
    // RepoWatchModeDont explicit don't auto-watch
    RepoWatchModeDont // 2
    // RepoWatchModeAuto watch repository (from AutoWatchOnChanges)
    RepoWatchModeAuto // 3

```

Reply:
but now IssueWatchModeNone will have no function at all!

Reply:
True, but we can make `RepoWatchModeNormal` start with `iota+1` anyway.

Reply:
@guillep2k done